src/utils/IdealGraphVisualizer/Coordinator/src/main/java/com/sun/hotspot/igv/coordinator/OutlineTopComponent.java

```diff
@@ -205,6 +217,19 @@ protected boolean requestFocusInWindow(boolean temporary) {
 
     @Override
     public void resultChanged(LookupEvent lookupEvent) {
+        // Highlight the focused graph, if available, in the outline.
+        if (result.allItems().isEmpty()) {
+            return;
+        }
+        final InputGraphProvider p = LookupHistory.getLast(InputGraphProvider.class);
+        if (p == null) {
+            return;
+        }
+        try {
+            manager.setSelectedNodes(new GraphNode[]{FolderNode.getGraphNode(p.getGraph())});
```

Comment:
Do we need to consider that FolderNode.getGraphNode() returns null? `setSelectedNodes` will throw `IllegalArgumentException` if input is null?

Reply:
`FolderNode.getGraphNode()` should never return null, so if that happens I think the best option is to warn the user and log the `IllegalArgumentException` exception thrown by `setSelectedNodes()`, which is what `Exceptions.printStackTrace()` does. See https://bits.netbeans.org/12.6/javadoc/org-openide-explorer/org/openide/explorer/ExplorerManager.html#setSelectedNodes-org.openide.nodes.Node:A- and https://bits.netbeans.org/12.6/javadoc/org-openide-util/org/openide/util/Exceptions.html#printStackTrace-java.lang.Throwable-.